samples/subsys/power/power_mgr/src/main.c

```diff
@@ -11,6 +11,7 @@
 #include <string.h>
 #include <device.h>
 #include <gpio.h>
+#include "policy/pm_policy.h"
```

Comment:
hmm, is this a private API?

Reply:
yes @nashif 

I will move this to include/power.h in my upcoming PR.

Reply:
can you submit that PR first?